packages/design-picker/src/components/index.tsx

```diff
@@ -17,9 +17,7 @@ import {
 import { DesignPickerCategoryFilter } from './design-picker-category-filter';
 import MShotsImage from './mshots-image';
 import type { Categorization } from '../hooks/use-categorization';
-export { default as MShotsImage } from './mshots-image';
```

Comment:
We've got to keep exporting this because it's used in the [new page modal](https://github.com/Automattic/wp-calypso/blob/2b61c991f03bbc7ac961f75b183bcd8feba2017d/packages/page-pattern-modal/src/components/pattern-selector-item.tsx#L1) and a [DIFM component](https://github.com/Automattic/wp-calypso/blob/2b61c991f03bbc7ac961f75b183bcd8feba2017d/client/signup/steps/difm-design-picker/demo-tile.tsx#L2).

Reply:
Actually, it exports from [here](https://github.com/Automattic/wp-calypso/blob/2b61c991f03bbc7ac961f75b183bcd8feba2017d/packages/design-picker/src/index.ts#L3), so we don't need this anymore.